Makefile

```diff
@@ -14,13 +14,13 @@ COMBO_VERSION :=  $(shell git describe --match 'v[0-9]*' --tags --always)
 export KUBERNETES_VERSION=v0.22.2
 
 # Container build options
-export IMAGE_REPO=quay.io/operator-framework/combo-operator
-export IMAGE_TAG=latest
-IMAGE=$(IMAGE_REPO):$(IMAGE_TAG)
+export IMAGE_REPO ?= quay.io/operator-framework/combo-operator
+export IMAGE_TAG ?= latest
```

Comment:
This was necessarily as the release GHA workflow uses the `release` Makefile target, and we were always using the latest value despite overriding this value during that workflow.